geomstats/geometry/invariant_metric.py

```diff
@@ -708,7 +707,7 @@ def metric_matrix(self, base_point=None):
         jacobian = self.group.jacobian_translation(
             point=base_point, left_or_right=self.left_or_right)
 
-        inv_jacobian = GeneralLinear.inverse(jacobian)
+        inv_jacobian = gs.linalg.inv(jacobian)
```

Comment:
Which logic do we want to use with the linear algebraic operations? We prefer using `Matrices.mul`, `Matrices.transpose`, etc - but we recommend `gs.linalg.inv` instead of `GeneralLinear.inverse`?

Reply:
No in general we can use GeneralLinear.inverse, but it lead to circular imports in this case